pyanaconda/ui/gui/spokes/lib/accordion.py

```diff
@@ -203,7 +203,7 @@ def removeSelector(self, selector):
 
     def _mountpointType(self, mountpoint):
         if not mountpoint or mountpoint in ["/", "/boot", "/boot/efi", "/tmp", "/usr", "/var",
-                                            "biosboot", "prepboot", "swap"]:
+                                            "swap", "PPC PReP Boot", "BIOS Boot"]:
```

Comment:
Are you sure this works with partitions/devices created by users in custom partitioning? AFAICT we still expect them to type in/choose ``prepboot`` as the mountpoint not ``PPC PReP Boot``.